extra/apm_guide_landing.html

```diff
@@ -0,0 +1,36 @@
+<style>
+  #content > .ulist .itemizedlist {
+    visibility: hidden;
+  }
+</style>
+
+<div class="w-100 mb-5">
+  <div class="important admon">
+    <div class="icon"></div>
+      <div class="admon_content">
+      <p class="font-weight-bold">
+        Starting with version 8.12, the content from the APM guide has been moved to
+        the <a class="font-weight-bold" href="/guide/en/observability/current/apm.html">Observability guide</a>.
+      </p>
+    </div>
+  </div>
+</div>
+<div class="w-100 mb-5">
+  <h2>Previous versions:</h2>
+  <ul>
+    <li><a href="/guide/en/apm/agent/guide/8.11/index.html">APM Guide: 8.11</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.10/index.html">APM Guide: 8.10</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.9/index.html">APM Guide: 8.9</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.8/index.html">APM Guide: 8.8</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.7/index.html">APM Guide: 8.7</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.6/index.html">APM Guide: 8.6</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.5/index.html">APM Guide: 8.5</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.4/index.html">APM Guide: 8.4</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.3/index.html">APM Guide: 8.3</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.2/index.html">APM Guide: 8.2</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.1/index.html">APM Guide: 8.1</a></li>
+    <li><a href="/guide/en/apm/agent/guide/8.0/index.html">APM Guide: 8.0</a></li>
+    <li><a href="/guide/en/apm/agent/guide/7.17/index.html">APM Guide: 7.17</a></li>
+    <li><a href="/guide/en/apm/agent/guide/7.16/index.html">APM Guide: 7.16</a></li>
+  </ul>
+</div>
```

Comment:
I updated this and hard-coded previous versions so we could still keep publishing `8.12` and `master` to keep the temporary redirects up (and satisfy the link checker) without linking to them here.

<img width="1459" alt="Screenshot 2024-01-03 at 2 27 13 PM" src="https://github.com/elastic/docs/assets/10479155/4e16b66a-3efb-44f4-a3c6-d1a04802aa2a">